src/lib/components/MenuBackground.svelte

```diff
@@ -80,8 +62,11 @@
   .on-chain {
     // We do not want to display the logo "on chain" behind the menu items
     display: none;
-    padding: 0 0 var(--padding-3x);
+    margin-bottom: var(--padding-3x);
+    width: 60%;
```

Comment:
not sure if `width` and `opacity` should be defined here because if a developer provide their own "one liner" they will also inherits it.

maybe those style should be applied on the default onliner what do you think?

Reply:
Ah yeah of course! I could a) use inline styles or b) have a special class on our own logo. What would you like better?

Reply:
oh, but they would not use the on-chain class for their own-liner no? I think we are safe.